src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -1126,5 +1228,180 @@ public void Dispose()
             Dispose(true);
             GC.SuppressFinalize(this);
         }
+
+        // Maps GLFW's key enum to our key enum.
+        // There's a few gaps here and there since this is a direct array.
+        // Those default to Unknown.
+        private static readonly Key[] GlfwKeyMapping = GenerateGlfwKeyMapping();
```

Comment:
As discussed on Discord, it would require us to make `Key` match GLFW's key enum. This would reduce the packing efficiency of it and make `KeyboardState` a LOT heavier.

It's also kind of iffy since the key enum is supposed to work on other platforms like mobile too, which GLFW does not.